data/MI/incentives.json

```diff
@@ -1278,49 +1173,49 @@
     "program": "mi_indianaMichiganPower_i&MMarketplace",
     "amount": {
       "type": "dollar_amount",
-      "number": 155,
-      "minimum": 50,
-      "maximum": 155
+      "number": 25,
+      "minimum": 15,
+      "maximum": 80
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Get rebates up to $155 on smart thermostats via the I&M Marketplace."
+      "en": "$25 rebate for smart thermostats and $15 rebate for Wi-Fi thermostats. Limit 2 of each kind per account lifetime. Household must have electric heat."
     },
     "bonus_available": true
   },
   {
-    "id": "MI-61",
+    "id": "MI-65",
```

Comment:
this diff is a little confusing. changes:
1. omit MI-61-63. I can't find any reference to them on I&M's website so I think they must have expired in 2023
2. added a couple new incentives